server/server-stats.cpp

```diff
@@ -724,6 +729,7 @@ void ServerStats::write_stats_to(std::ostream &os, bool add_worker_pids) const n
      << "RSS_max\t" << get_sum(general_vm, job_vm, master_vm, VMStat::Key::rss_peak_kb) << "Kb\n"
      << "tot_queries\t" << total_queries[QueriesStat::Key::incoming_queries].load(std::memory_order_relaxed) << "\n"
      << "tot_script_queries\t" << total_queries[QueriesStat::Key::outgoing_queries].load(std::memory_order_relaxed) << "\n"
+     << "tot_script_long_queries\t" << total_queries[QueriesStat::Key::outgoing_long_queries].load(std::memory_order_relaxed) << "\n"
```

Comment:
Let's skip this stats here.. and below with worker pid..

This is a sort of legacy, no need to support them..

Reply:
removed